src/main/java/tutoraid/ui/LessonCard.java

```diff
@@ -0,0 +1,56 @@
+package tutoraid.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import tutoraid.model.lesson.Lesson;
+
+/**
+ * A UI component that displays information of a {@code Lesson}.
+ */
+public class LessonCard extends Card<Lesson> {
+
+    private static final String FXML = "LessonListCard.fxml";
+
+    private static final String LESSON_NAME_LABEL = "Lesson";
+    private static final String TIMING_LABEL = "Timing";
+    private static final String PRICE_LABEL = "Price";
+    private static final String CAPACITY_LABEL = "Capacity";
+    private static final String STUDENTS_LABEL = "Students";
```

Comment:
Should these variable names be prefixed with LABEL instead, since associated constants should have a common prefix? e.g. `LABEL_LESSON_NAME`